tests/src/dir.targets

```diff
@@ -120,7 +119,8 @@
 
   <Target Name="CopyNativeProjectBinaries">
      <ItemGroup>
-        <NativeProjectBinaries Include="$(NativeProjectOutputFolder)\**\*.*" />
+        <NativeProjectBinaries Include="$(NativeProjectOutputFolder)\**\*.*" Condition="'$(BuildOS)' == 'Windows_NT'"/>
+        <NativeProjectBinaries Include="$(ProjectDir)\..\bin\Product\$(OSPlatformConfig)\bin\*.*" Condition="'$(BuildOS)' != 'Windows_NT'"/>
```

Comment:
We are doing this because building of natives is part of CoreCLR build system, not tests, so we have different location. Please keep in mind this scripts are also working on Linux, whose toolset in fact contains lots of bugs .